lib/java/ydoc-api/src/main/java/module-info.java

```diff
@@ -0,0 +1,3 @@
+module org.enso.ydoc.api {
+  exports org.enso.ydoc.api;
```

Comment:
- There is no need for a new API module.
- there already is [YdocServerApi](https://github.com/enso-org/enso/pull/14438/files#diff-0803ce07fcddab22935457df18ed5ecbb770e7fe0b26b00117e5d2c370f752a8L7)
- please put the new interfaces there as inner classes of `YdocServerApi`.